templates/ecs_tasks/consignment_export.json.tpl

```diff
@@ -15,21 +19,25 @@
         "value": "${clean_bucket}"
       },
       {
-        "name": "OUTPUT_BUCKET",
```

Comment:
Why are these bucket environment variables not needed any more?

Reply:
They're set in the step function. We're running the same task for both processes, the only difference being the run command and the export bucket so they're added in there instead. It reduces the amount of new stuff I had to create.